README.md

```diff
@@ -2,9 +2,9 @@
   <img alt="NSIDC logo" src="https://nsidc.org/themes/custom/nsidc/logo.svg" width="150" />
 </p>
 
-# nsidc-metgen
+# MetGenC
 
-`nsidc-metgen` via the MetGenC utility, enables Operations staff and data
+The `MetGenC` toolkit enables Operations staff and data
```

Comment:
@eigenbeam Here's my argument for just using the term "MetGenC" in this context: The package name `nsidc-metgenc` includes `nsidc` for name-spacing purposes. If we want to include `nsidc` in this README text then I think it should read `NSIDC MetGenC` (similar to `SIPS MetGen`). Thoughts?

Reply:
Yep, sounds good!